src/integration-test/tests/citizen/claim/steps/claim.ts

```diff
@@ -339,67 +339,6 @@ export class ClaimSteps {
     this.enterClaimEvidence()
   }
 
-  async makeAHwfClaimAndNavigateUpToPayment (I: I) {
```

Comment:
Removing this test as the same steps are executed in 
Scenario('Submit Claim via HWF Reference... @citizen', { retries: 3 }, async (I: I) => {
  await claimSteps.makeAHwfClaimAndSubmit(I)
})